app/src/main/java/fr/free/nrw/commons/nearby/Place.java

```diff
@@ -26,12 +26,14 @@
     private final String category;
     public final String pic;
     public final String destroyed;
+    public final String endTime;
+    public final Boolean exists;
```

Comment:
Please explain what will this exists will hold with a comment. do not forget to mention that some places may have destroyed and some may have end time. It is not guaranteed that we have both of them on the same place even if the place does not exist.

Reply:
Done!